lib/ffmpeg/util.py

```diff
@@ -66,3 +66,40 @@ def map_best_hw_format(format, hwformats):
   return mapformat(
     match_best_format(
       format, set(hwformats) & set(get_supported_format_map().keys())))
+
+@memoize
+def ffmpeg_cmd():
+  from lib.platform import info
+  if info()["os"] == "windows":
+    return 'ffmpeg.exe'
+  else:
+    return 'ffmpeg'
+
+@memoize
+def ffprobe_cmd():
+  from lib.platform import info
+  if info()["os"] == "windows":
+    return 'ffprobe.exe'
+  else:
+    return 'ffprobe'
+
+@memoize
+def wsl_path_convert(file_path):
+  from lib.platform import info
+  if info()["os"] == "windows":
+    # for the WSL will automation mount windows file system to /mnt/DRIVER/file/path
+    # windows execute file need to access really windows file path is DRIVER:\file\path
+    wsl_file_path = file_path.replace('/mnt/', '')
+    wsl_file_path = wsl_file_path.replace('/', '\\\\')
+    if wsl_file_path[1] == '\\':
+      wsl_file_path = wsl_file_path[0] + ':' + wsl_file_path[1:]
```

Comment:
Can we use the `wslpath` command here?
```python
windows_path = call(f"wslpath -w {file_path}")
```

Reply:
Well, honestly I think it would be better to distinguish between the two... that is, platform.info()["os"] should return "wsl" when running in WSL and "windows" when running natively.

Reply:
Another point... your method/variable naming conventions already imply "wsl".  We should avoid that.

Reply:
So I did some experimenting.  IMO, we should return "wsl" in `info["os"]` for WSL environment and use the `wslpath` command to convert the filepath in that case.  The benefit is that the files can exist in a WSL directory (i.e. non-native) or a windows native directory.  And the `wslpath` command will resolve it properly for both so that native windows executables can work (e.g. ffmpeg.exe).

For example, from WSL shell:

```shell
wslpath -w /home/user/path/to/file # returns "\\wsl$\Ubuntu-20.04\home\user\path\to\file"
wslpath -w /mnt/c/path/to/file # returns "C:\path\to\file"
```

The reason that the asset paths are in "posixpath" format in WSL is because we use `os.path.join` in the config files (as we should).  If we were actually running from native windows, the `os.path.join` would return "ntpath" formats.

Hence, we must distinguish the OS between native windows and WSL... and we won't support native windows, yet.